.gitignore

```diff
@@ -3,3 +3,6 @@
 *.zip
 lanthan-driver.log
 package-lock.json
+*.swo
+*.swp
+tags
```

Comment:
for "*.swo" and "*.swp" it's for vim user who doesn't disable swap file, it's really useless to keep it in the repo
for "tags" it's a file that is generated by ctags, it's used to navigate between function, it's also useless to keep it in the repo and if it's not correctly generated, ctags will search function in node_modules and create a pretty heavy file

Reply:
I still feel like these *user-specific* files can be better put in a global gitignore file for that user. E.g. I have a global gitignore under `~/.config/git/ignore` where I put things I want to ignore, personally, for all repositories I touch. (See man page for `git-config`.) This would be better put in that file, because it doesn't really have anything to do with files that are generated by the project itself, i.e. build artifacts, node_modules, or whatever other things that every contributor to *this repo* would want to ignore. This on the other hand is due to the editor choice of select people only.

I hope that makes sense. If you don't want to ignore ctags globally for all your repos, but only for this repo, you can add a line `ctags` in `.git/info/exclude`, or simply not check the file into the index (will be caught during code review anyway, surely).